aboutmejs.js

```diff
@@ -0,0 +1,86 @@
+ // 
+ const page = document.querySelector('body');
+ page.style.fontFamily='arial';
+
+
+
+
+// 
+document.getElementById('nickname').innerText = 'Munira AlShuhail';
+document.getElementById('favorite').innerText = 'None.. I hate animals!';
+document.getElementById('hometown').innerText = 'Saudi Arabia, AlAhsa';
+
+
+//
+
+document.getElementById('nickname');
+nickname.style.color='rebeccapurple';
+document.getElementById('favorite');
+favorite.style.color='rebeccapurple';
+document.getElementById('hometown');
+hometown.style.color='rebeccapurple';
+
+//
+const listItem = document.querySelector('ul');
+listItem.style.color='rebeccapurple';
+
+
+
+
+// var items = document.getElementsByTagName('li');
+// for (var i = 0; i>= items.length; i++) {
+//     items[i].className = 'listitem';
+//  }
+
+
+
+
+//
+
+var myPic = document.createElement('img');
+    myPic.src = 'batman.jpg';
+    document.body.appendChild(myPic);
+
+
+    // The book list 
+
+    // var books = [
+    //     {
+    //       title: "The Design of Everyday Things",
+    //       author: "Don Norman",
+    //       alreadyRead: false
+    //     },
+    //     {
+    //       title: "The Most Human Human",
+    //       author: "Brian Christian",
+    //       alreadyRead: true
+    //     },
+    //     {
+    //       title: "In Search of Lost Time",
+    //       author: "Marcel Proust",
+    //       alreadyRead: true
+    //     },
+    //     {
+    //       title: "Ulysses",
+    //       author: "James Joyce",
+    //       alreadyRead: true
+    //     },
+    //     {
+    //       title: "The Great Gatsby",
+    //       author: "F. Scott Fitzgerald",
+    //       alreadyRead: true
+    //     }
+    //   ];
+
+    //   <h1>My Book List</h1>
+
+    //   for (var i = 0; i < books.length; i++) {
+    //     var bookP = document.createElement('p');
+    //     var bookDescription = document.createTextNode(books[i].title + ' by ' + books[i].author);
+    //     bookP.appendChild(bookDescription);
+    //     document.body.appendChild(bookP);
```

Comment:
This was working, not sure why it was commented? You were only missing the closing bracket of your loop `}`